db/setup/schema.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE ft_transcendence;
+
+\c ft_transcendence;
+
+CREATE TABLE IF NOT EXISTS users(
+    UUID        UUID DEFAULT gen_random_uuid()  PRIMARY KEY,
```

Comment:
I've gone with randomly generated UUID with a function already provided by postgreSQL -> https://www.cybertec-postgresql.com/en/uuid-serial-or-identity-columns-for-postgresql-auto-generated-primary-keys/

There are many more methods but this seemed the best to me

Reply:
Is it possible (even if it's VERY unlikely) that we will get duplicated UUID?

Reply:
I think it is not very likely. But, with large set of data it may be possible -> https://stackoverflow.com/questions/47172263/many-duplicate-values-when-using-postgres-uuid-generate-v4

For our usecase I'd say it is good enough :)